administrator/components/com_media/resources/scripts/mediamanager.js

```diff
@@ -20,6 +21,7 @@ import store from './store/store';
 Vue.use(Translate);
 
 // Register the vue components
+Vue.component('media-adapter', Adapter);
```

Comment:
Not so happy with the naming "adapter". 
An adapter is just a wrapper for the media provider api.
We should not use that in any part of the frontend. We can discuss this in our next meeting.